### PR TITLE
add ability to list conversations by customer

### PR DESCRIPTION
**Pull request by AlexeyMK**
option 3 on
http://developer.helpscout.net/help-desk-api/conversations/list/

previously, only options 1 and 2 were implemented

### CODE REVIEW

**Comment by AlexeyMK** [comment]
@soul2197

File: lib/helpscout/client.rb [hunk 1]
```diff
@@ -772,6 +772,92 @@ def conversations_in_folder(mailboxId, folderId, status, limit=0, modifiedSince)
...
+
+      begin
+        options["page"] = page
+        items = Client.request_items(@auth, url, options)
+        items.each do |item|
+          conversations << Conversation.new(item)
+        end
+        page = page + 1
+      rescue StandardError => e
+        puts "List Conversations Request failed: #{e.message}"
```
Comment (soul2197) [inline lib/helpscout/client.rb]: you should probably raise here

**Comment by soul2197** [comment]
lgtm besides my comment

File: lib/helpscout/client.rb (hunk 1 above)
Comment (AlexeyMK) [inline lib/helpscout/client.rb]: lol, indeed. I was copy-pasting but that seems pretty off.

**Pull request merged**